utils/luci-app-owm/luasrc/owm.lua

```diff
@@ -205,6 +205,15 @@ function showmac(mac)
 	return mac
 end
 
+function get_position()
+	local position = {}
+	uci:foreach("system", "system", function(s) --owm
+		position['latitude'] = tonumber(s.latitude) --owm
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

Am 14.02.16 um 18:58 schrieb Sven Roederer:
>> @@ -205,6 +205,15 @@ function showmac(mac)
>>  	return mac
>>  end
>>  
>> +function get_position()
>> +	local position = {}
>> +	uci:foreach("system", "system", function(s) --owm
>> +		position['latitude'] = tonumber(s.latitude) --owm
> 
> So any preference to keep them or to remove these "--owm" comments?
Remove and make a hint where is data structure documented
https://github.com/freifunk/openwifimap-api#standard-fields

-----BEGIN PGP SIGNATURE-----
Version: GnuPG/MacGPG2 v2
Comment: GPGTools - http://gpgtools.org

iEYEARECAAYFAlbAwYcACgkQr9m0OkMZoqSd8wCdErBuRriia5i0DXCQPG5CQved
KHUAnRm978WZcr37vy5OxSlNG9NKogL0
=pqOD
-----END PGP SIGNATURE-----